packages/commons-ui-core/src/NavList/NavIcon.js

```diff
@@ -0,0 +1,53 @@
+import { Grid } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+import ImageButton from "../ImageButton/index";
+
+const ImageIcon = styled(ImageButton)({
+  boxShadow: "none",
+  textTransform: "capitalize",
+  fontSize: 48,
+  margin: 24,
+  backgroundColor: "transparent",
+  borderColor: "none",
+  "&:hover, &:focus, &:active, &:focus-within": {
+    backgroundColor: "transparent",
+    borderColor: "transparent",
+    boxShadow: "none",
+  },
+});
+
+function NavIcon({ links }) {
```

Comment:
I don't think we need this component... Twitter is just an icon and MUI already has [`IconButton`](https://mui.com/material-ui/react-button/#icon-button) we can use for these special cases